app/scenes/Login/Notices.tsx

```diff
@@ -71,6 +71,12 @@ export default function Notices() {
           admin.
         </NoticeAlert>
       )}
+      {notice === "invite-required" && (
```

Comment:
the content of this notice originates with 
```
new Error("Invite required");
```
I'm not thrilled about this, as it seems a bit brittle; open to having a more rigid taxonomy for error types

Reply:
You should create an error in `server/errors` and then it'll be easy to use the existing logic…